kahuna/public/js/components/gr-crop-image/gr-crop-image.js

```diff
@@ -2,24 +2,35 @@ import angular from 'angular';
 
 import template from './gr-crop-image.html!text';
 
+import '../../services/api/leases';
+
 export const cropImage = angular.module('gr.cropImage', [
 
 ]);
 
 cropImage.controller('grCropImageCtrl', [
-    '$scope', 'mediaCropper', 'onValChange',
-    function ($scope, mediaCropper, onValChange) {
+    '$scope', 'mediaCropper', 'onValChange', 'leaseService', '$rootScope',
+    function ($scope, mediaCropper, onValChange, leaseService,  $rootScope) {
         let ctrl = this;
 
         function updateState () {
-            mediaCropper.canBeCropped(ctrl.image).then(croppable => {
-                ctrl.canBeCropped = croppable;
+            leaseService.allowedByLease(ctrl.image).then(allowed => {
```

Comment:
You might want to prefix with a verb e.g. `isAllowedByLease` to indicate that you're asking.